datastream-server/src/main/java/com/linkedin/datastream/server/CoordinatorEventBlockingQueue.java

```diff
@@ -38,7 +38,7 @@
 class CoordinatorEventBlockingQueue implements MetricsAware {
 
   private static final Logger LOG = LoggerFactory.getLogger(CoordinatorEventBlockingQueue.class.getName());
-  private static final Set<BrooklinMetricInfo> METRIC_INFOS = ConcurrentHashMap.newKeySet();
```

Comment:
There are some metrics tests that you can add to validate if the metrics are emitted or not. Please add similar tests for this one.

Reply:
The existing tests are passing - [testRegistersMetricsCorrectly()](https://github.com/linkedin/brooklin/blob/c317f12b5106c8d78da8affb16dbc7bcf0d1c03d/datastream-server-restli/src/test/java/com/linkedin/datastream/server/TestCoordinator.java#L195) - metrics are not explicitly validated. The issue here is **InGraphs**. The link in the description was so that reviewers could see that registration in InGraphs requires explicit registration.